### ISSUE TITLE
FISMrmseRecommender类中的一个错误

### ISSUE THREAD

**Issue by hegongshan** [guoguibing/librec#316]
在源码第164行中，
loss += itemBiasReg * bu * bu;
虽然itemBiasReg和userBiasReg的初始值相等，但正确的写法应该是 loss += userBiasReg * bu * bu;
